examples/builders/amazon-builder.json

```diff
@@ -1,22 +1,12 @@
 {
   "builders": [
     {
-      "type": "ami",
+      "type": "Amazon AWS",
       "account": {
-        "type": "ami",
-        "name": "My AWS Account",
-        "accountNumber": "111122223333",
-        "x509PrivateKey": "/home/joris/accounts/aws/pk509.pem",
```

Comment:
It seems that we only use a reference to an already created credaccount now. I would prefer to have all the mandatory information needed to create a cloud account